qutebrowser/config/configdata.yml

```diff
@@ -1358,6 +1358,11 @@ tabs.indicator.padding:
 tabs.width.pinned:
   deleted: true
 
+tabs.pinned.shrink:
+  default: true
+  type: Bool
+  desc: Shrink pinned tabs down to their contents
```

Comment:
Nitpick: Could you add a period (`.`) to the end of the description (see https://github.com/qutebrowser/qutebrowser/commit/a55d12bf708a9178e81f42948229eeee2da5735e)? :wink:

Reply:
Thanks for the catch! :)